review previous portuguese docs (#17)
stack-spot/graphene-java-api-stack: metrics-plugin/docs/usage.md

#### **Requirements**

> To use this plugin, it's need to have an application based on [Stack Web API com Spring](https://github.com/stack-spot/graphene-java-api-stack/tree/main/spring-web-api-template) and created with `StackSpot CLI`, which you can download [**here**](https://stackspot.com/).

Reviewer: ```suggestion
- You need an application based on [Stack Web API com Spring](https://github.com/stack-spot/graphene-java-api-stack/tree/main/spring-web-api-template) and created with [**STK CLI**](https://stackspot.com/).

```

> It's also need to install the itens below:
> - [**StakSpot CLI**](https://docs.stackspot.com/v3.0.0/os-cli/installation/)

Reviewer: ```suggestion
See below the items you need to have installed: 
- [**STK CLI**](https://docs.stackspot.com/v3.0.0/os-cli/installation/)
```
